crates/swc_ecmascript/README.md

```diff
@@ -0,0 +1,3 @@
+# Deprecated
```

Comment:
Digging dep tree, swc_ecmascript was not being referenced across the internal pkgs, so attempt to make one-last publish to include readme then improve swc_core for the external usecase.